src/Bonsai.ML.Torch/LinearAlgebra/SVD.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.ComponentModel;
+using System.Reactive.Linq;
+using static TorchSharp.torch;
+
+namespace Bonsai.ML.Torch.LinearAlgebra
+{
+    /// <summary>
+    /// Computes the singular value decomposition (SVD) of a matrix.
+    /// </summary>
+    [Combinator]
+    [Description("Computes the singular value decomposition (SVD) of a matrix.")]
+    [WorkflowElementCategory(ElementCategory.Transform)]
+    public class SVD
```

Comment:
```suggestion
    public class Svd
```

As per framework design guidelines, 3-letter acronyms should have only the first letter capitalized.

Alternatively we could be explicit and call it `SingularValueDecomposition`. A good source of inspiration for explicit naming for all these things can be found in the Math.NET documentation, for example for decompositions:

https://iridium.mathdotnet.com/api/MathNet.Numerics.LinearAlgebra/index.htm